portfolio/src/main/java/com/google/sps/servlets/DeleteCommentsServlet.java

```
@@ -93,7 +94,7 @@ public Long tryParseLong(String str) {
         try {
             return Long.parseLong(str);
         } catch (NumberFormatException e) {
-            return null;
+            return -1;
```

It is better to throw an exception and catch it from the caller